errors/errors.go

```diff
@@ -68,13 +68,13 @@ func (e *Error) Error() string {
 	message = e.Message
 	if message == "" {
 		switch e.Type {
+		case NotImplemented:
```

Comment:
This prevents the other error types from falling back to the notimplemented error. This was a bug from the errors PR.